app/uk/gov/hmrc/claimvatenrolmentfrontend/views/errorPages/third_attempt_lockout_page.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.claimvatenrolmentfrontend.views.html.helpers._
+@import uk.gov.hmrc.claimvatenrolmentfrontend.views.html.templates.layout
+
+
+@this(layout: layout,
+h1: h1,
+p: p,
+govUkHeader: GovukHeader,
+govukButton: GovukButton
+)
+
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(Some(titleNoForm(messages("third_attempt_lockout_page.title")))) {
+
+@h1(messages("third_attempt_lockout_page.heading"))
+
+@p {
+@messages("third_attempt_lockout_page.line_1")
+}
+
+@p {
+@messages("third_attempt_lockout_page.line_2")
+}
+
+@p {
+@messages("third_attempt_lockout_page.line_3")
+}
+
+<div class="govuk-button-group">
+@govukButton(Button(
+attributes = Map("id" -> "Go back to your Business Tax Account"),
```

Comment:
id attributes should be lowercase and kebab case e.g. 'back-to-bta' same as you would for any css style id (which it is) or class.
We're not applying any styles to the id so is a bit redundant but we can leave it in with the correct case, please

Reply:
If you are using intelliJ, please can you also apply the automatic formatter to each of your files to keep the indented and consistent by using `Code > Reformat Code` or using the shortcut `command` + `option` + `L`

Reply:
It's also good practice to leave an empty line at the end of the file due to some tools and conversions requiring this to read it properly so has just become the standard and will fail some team's formatters if not the case.
That's why github always shows the red warning at the end of the file if it's not there. Please could you add it to each of these files.